src/components/judgments/Judgments.js

```diff
@@ -1,12 +1,11 @@
 import React, {Component} from "react";
 import {Header, Button, Divider} from "semantic-ui-react";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'Divider'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)